apps/launchpad/app/lib/hooks/useGetQuestions.tsx

```diff
@@ -0,0 +1,26 @@
+import logger from '@lib/utils/logger'
+import { useQuery } from '@tanstack/react-query'
+
+export function useGetQuestions() {
+  return useQuery({
+    queryKey: ['get-questions'],
+    queryFn: async () => {
+      try {
+        const response = await fetch('/resources/get-questions')
```

Comment:
Awesome. Glad this pattern worked. We should use this more often / have an example of this use case.